package.json

```diff
@@ -19,5 +19,10 @@
         "lerna": "3.20.2",
         "tslint": "6.0.0",
         "start-server-and-test": "1.10.8"
+    },
+    "dependencies": {
+        "JSONStream": "^1.3.5",
+        "mongodb": "^3.5.5",
+        "typescript": "^3.8.3"
     }
```

Comment:
These dependencies should not be added at the root of the repository. Please place them in the appropriate package